packages/spruce-node/index.ts

```diff
@@ -652,7 +652,7 @@ export default class Sprucebot {
 
 		// not found, create it
 		if (!meta) {
-			meta = await this.createMeta(key, value, Array.from(arguments)[2])
```

Comment:
@liquidg3 @kengoldfarb can you sanity check this? At a blush `Array.from(arguments)[2]` would always get options...  but maybe I'm missing something.

Reply:
yes, this was legacy when there was magic on the signature of that method. your fix is 100%.